bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -21,7 +21,50 @@ spec:
         echo -n "."
         sleep 1
       done
+  - name: wait-for-other-host-port
+    terminationMessagePolicy: FallbackToLogsOnError
+    image: ${CLUSTER_POLICY_CONTROLLER_IMAGE}
+    imagePullPolicy: IfNotPresent
+    command: ['/usr/bin/timeout', '30', "/bin/bash", "-c"]
+    args:
+    - |
+      echo -n "Waiting for port :10258 to be released."
```

Comment:
this is the cloud controller manager port.  Choose something in a different range.

Reply:
fixed, it's 10357 now : )